main/index.js

```diff
@@ -63,6 +66,12 @@ const ctx = {
     ipcMain.emit(ipcMainEvents.ACTIVITY_LOGGED, activityLog.getAllEntries())
   },
 
+  getTotalJobsCompleted: () => jobStats.getTotalJobsCompleted(),
+  setModuleJobsCompleted: (moduleName, count) => {
+    jobStats.setModuleJobsCompleted(moduleName, count)
+    ipcMain.emit(ipcMainEvents.JOB_STATS_UPDATED, jobStats.getTotalJobsCompleted())
```

Comment:
```suggestion
    ipcMain.emit(ipcMainEvents.JOB_STATS_UPDATED, {
      totalJobsCompleted: jobStats.getTotalJobsCompleted()
    })
```

I think this would be nice to make way for further stats, also this way the content matches the event name more

Reply:
I have considered emitting stats as an object too. But then, we would also need to change the getter function for consistency.

In the end, I decided to keep things simple. We have complete control of the backend & frontend, and they are both moving together in a lock step. Reworking stats from a number to an object can be easily done at any time.

[YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)